packages/neutrino-middleware-image-loader/index.js

```diff
@@ -3,13 +3,14 @@ const merge = require('deepmerge');
 module.exports = ({ config }, options) => {
   const { limit } = merge({ limit: 8192 }, options);
   const urlLoader = require.resolve('url-loader');
+  const svgUrlLoader = require.resolve('svg-url-loader');
 
   config.module
     .rule('svg')
     .test(/\.svg(\?v=\d+\.\d+\.\d+)?$/)
-    .use('url')
-      .loader(urlLoader)
-      .options({ limit, mimetype: 'image/svg+xml' });
+    .use('svgUrl')
```

Comment:
We can actually probably just leave this `.use` as `url`. It's still a URL "loader", even though the actual loader is different. The `.use` call just defines the ID of the loader.